awsclient/generate_names.go

```diff
@@ -1,27 +1,33 @@
 // Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
 // See License for license information.
 
-package aws
+package awsclient
 
 import (
 	"crypto/sha256"
 	"encoding/hex"
 	"fmt"
+	"os"
 	"strings"
 
 	"github.com/pkg/errors"
 
 	"github.com/mattermost/mattermost-plugin-apps/apps"
 )
 
-// getFunctionName generates function name for a specific app
-// name can be 64 characters long.
-func getFunctionName(appID apps.AppID, version apps.AppVersion, function string) (string, error) {
-	if len(appID) > appIDLengthLimit {
-		return "", errors.Errorf("appID %s too long, should be %d bytes", appID, appIDLengthLimit)
+const LambdaFunctionFileNameMaxSize = 64
+const AppIDLengthLimit = 32
+const VersionFormat = "v00.00.000"
+const StaticAssetsFolder = "static/"
```

Comment:
I believe we have them in `provision_app.go` as well (and moved to another file in https://github.com/mattermost/mattermost-plugin-apps/pull/93/files)

Reply:
Yes, I moved provision_app into `appsctl`, but the name generation functions are needed in the `server` also. Technically, these constants do not need to be public, but it's nice for godoc purposes IMO.